feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -159,7 +159,7 @@ class PreviewViewModel @Inject constructor(
             && previewUiState.value.currentCameraSettings.isFrontCameraAvailable
         ) {
 
-            //update viewmodel value
+            //update view model value
```

Comment:
I think we can just get rid of this comment.  It's inaccurate since we're updating the uiState here.